content/docs/tasks/security/vault-ca/index.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Vault CA integration
+description: Tutorial on how to plug in a Vault CA for issuing certificates in Istio.
+weight: 10
+keywords: [security,certificate]
+---
+
+This tutorial walks you through an example to plug in a Vault CA for issuing
+certificates in Istio.
+
+## Before you begin
+
+* Create a new Kubernetes cluster to run the example in this tutorial.
+
+* On GKE, you may set the value of the `CLUSTER`, `ZONE`, and `PROJECT`
+environmental variables based on your GKE cluster and run the following
+`gcloud` command to fetch the credentials for your cluster.
+
+    {{< text bash >}}
+    $ export CLUSTER=YOUR-CLUSTER-NAME
+    $ export ZONE=YOUR-CLUSTER-ZONE
+    $ export PROJECT=YOUR-GKE-PROJECT-NAME
+    $ gcloud container clusters get-credentials $CLUSTER --zone $ZONE --project $PROJECT
+    {{< /text >}}
+
+## Install Istio with SDS enabled
+
+1.  Use `git clone` to download the latest Istio source code.
+As the time of writing, the latest Istio code is on the release-1.1 branch
```

Comment:
Er, I'm not sure we need or want any of this GitHub gymnastic in customer documentation.

Documentation checked into the master branch of this repo will only be made available to customers when 1.1 is officially released. So I presume the customers won't need to do custom builds of anything in order to get the functionality, correct?

Reply:
Correct. The GitHub gymnastic  has been removed.